backend/index.ts

```diff
@@ -35,3 +36,53 @@ const port = 8000;
 app.listen(8000, () => {
     console.log(`Server started: http://localhost:${port}/`);
 }); 
+
+
+
+// Configure Strategy for Login
+
+// user is prompted to sign in with their username and password by rendering a form
+// defining a route
+
+app.get('/login/password',
+  function(req, res, next) {
+    res.render('login');
+  });
+
+app.post('/login/password',
+  passport.authenticate('local', { failureRedirect: '/login', failureMessage: true }),
+  function(req, res) {
+    console.log(req.body)
+    res.status(200).send("we logged in")
+  });
+
+var passport = require('passport');
+var LocalStrategy = require('passport-local');
+var crypto = require('crypto');
+var mysql = require('mysql12')
+
+// connecting to the database
+var connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    password: '',
+    database: 'wewrap_local'
```

Comment:
whoever gets to it first, you or @jesshe004, will want to move these fields into the `.env`!

Reply:
I'm a little bit confused as to how to move the list of these variables into .env and then properly relate back to them on the index.ts file when creating the mysql connection. I'd be willing to do this if Jess doesn't get to it first but I just need some clarification.

Reply:
here it will look like:
```
var connection = mysql.createConnection({
    host: process.env.DB_HOST,
    user: process.env.DB_USER,
    password: process.env.DB_PASSWORD,
    database: process.env.DB_NAME
  });

```

and in the .env

```
DB_HOST="localhost"
DB_USER="root"
DB_PASSWORD=""
DB_NAME="wewrap_local"

```

Reply:
Oooo this really helps understanding it thanks!